### PR TITLE
user_project_wrapper fix

### PR DESCRIPTION
**Pull request by marwaneltoukhy**
- Fixed user project wrapper configuration to change the power ring spacing to match the golden wrapper
  - This fixes the issue with precheck XOR
- Fixed CI to run precheck on the generated views from hardened user_project_wrapper using the CI, not running precheck on the views on the repo
- Updated precheck version on the CI

### CODE REVIEW

**Review by kareefardi — APPROVED** [review]

File: .github/workflows/user_project_ci.yml [hunk 1]
```diff
@@ -58,12 +57,17 @@ jobs:
...
 
       - name: Export PDK ROOT
-        run: |
-          echo "PDK_ROOT=/home/runner/work/pdk" >> $GITHUB_ENV
-      
-      - name: Export PDKPATH
+        run: echo "PDK_ROOT=/home/runner/work/pdk" >> $GITHUB_ENV
+
+      - name: Export PDK PATH
         run: echo "PDKPATH=/home/runner/work/pdk/sky130A" >> $GITHUB_ENV
```
Comment (mattvenn) [inline .github/workflows/user_project_ci.yml:63]: is this line still needed ? what does PDKPATH get used by, and shouldn't it be PDK_ROOT?

File: .github/workflows/user_project_ci.yml [hunk 2]
```diff
@@ -177,6 +149,9 @@ jobs:
       - name: Export PDK ROOT
         run: echo "PDK_ROOT=/home/runner/work/pdk" >> $GITHUB_ENV
 
+      - name: Export PDK PATH
+        run: echo "PDKPATH=/home/runner/work/pdk/sky130B" >> $GITHUB_ENV
```
Comment (mattvenn) [inline .github/workflows/user_project_ci.yml:153]: out of interest, why keep doing this and not set it in the envionment section of the workflow?

File: openlane/user_project_wrapper/config.json [hunk 3]
```diff
@@ -39,8 +39,8 @@
     "FP_PDN_CORE_RING_HWIDTH": 3.1,
     "FP_PDN_CORE_RING_VOFFSET": 12.45,
     "FP_PDN_CORE_RING_HOFFSET": 12.45,
-    "FP_PDN_CORE_RING_VSPACING": 12.45,
-    "FP_PDN_CORE_RING_HSPACING": 12.45,
+    "FP_PDN_CORE_RING_VSPACING": 1.7,
```
Comment (mattvenn) [inline openlane/user_project_wrapper/config.json:42]: remove this from the commit, I don't think it's related

**Review by mattvenn — CHANGES_REQUESTED** [review]

File: .github/workflows/user_project_ci.yml (hunk 1 above)
Comment (marwaneltoukhy) [reply .github/workflows/user_project_ci.yml:63]: Used for precheck @a-omla can comment

**Comment by marwaneltoukhy** [comment]
> I also think we should add a GF180 CI workflow.

Agree, but unrelated to this PR.

**Review by jeffdi — APPROVED** [review]

**Pull request merged**